AutowareAuto/src/control/trajectory_follower/src/mpc_utils.cpp

```diff
@@ -329,17 +329,21 @@ int64_t calcNearestIndex(
   if (traj.points.empty()) {
     return -1;
   }
-  const float64_t my_yaw = tf2::getYaw(self_pose.orientation);
+  const float64_t my_yaw = 0.0; //tf2::getYaw(self_pose.orientation);
```

Comment:
Is this correct? or did we forget to push commit.
If it is actually correct, let's explain why it needs to be this way.